app/ui/src/app/store/entity/entity.store.ts

```diff
@@ -72,7 +72,11 @@ export abstract class AbstractStore<
     return observableMerge(
       this._list,
       this.changeEvents.pipe(
-      debounceTime(500),
+        filter(event => {
+          log.debug(`got change event for kind: ${event.kind}, id: ${event.id}, this store is for kind: ${this.service.kind}`);
+          return event.kind === this.service.kind;
```

Comment:
Same comment for this line as below :-)

Reply:
@gashcrumb how we have it now is that every event triggers `list` of every resource, perhaps we should remove this subscriber all together and listen only to individual resource changes. Too drastic?

Reply:
no, this isn't shared for all resource, this is a base class for each resource, and we filter by kind in the constructor of this base class.  So this doesn't mean an event triggers fetching all resources, just for a given kind.